opentelemetry/build.gradle

```diff
@@ -15,11 +15,15 @@ dependencies {
             libraries.auto.value.annotations
 
     testImplementation project(':grpc-testing'),
+            project(':grpc-testing-proto'),
             project(':grpc-inprocess'),
+            project(':grpc-stub'),
+            project(':grpc-protobuf'),
             testFixtures(project(':grpc-core')),
             testFixtures(project(':grpc-api')),
             libraries.opentelemetry.sdk.testing,
             libraries.assertj.core // opentelemetry.sdk.testing uses compileOnly for assertj
+            libraries.protobuf.java
```

Comment:
This line does nothing, because there isn't a comma on the prior line. Delete it.